python/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -341,6 +341,10 @@ class HDBSCAN(UniversalBase, ClusterMixin, CMajorInputTagMixin):
     (and often will) lead to some points being assigned different
     cluster labels between the two implementations.
 
+    This estimator supports cuML's experimental device selection capabilities.
```

Comment:
_@beckernick I'll comment again just so it's in my "reviewed" list. Feel free to delete your comment after that._

Should we maybe automatically add this comment via a common decorator? Something like `@device_interop_enabled` ? This would also allow us to (maybe at a later stage) set some flag that helps with testing. This could of course be done in a follow-up to not block this PR.

Reply:
Thanks! I agree. Doc utilities for this kind of behavior is a great idea.

Reply:
It can be added to https://github.com/rapidsai/cuml/blob/branch-23.04/python/cuml/common/doc_utils.py, @beckernick could you open an issue to do it in a follow up? Thanks!